src/app/concern/concern.interfaces.ts

```diff
@@ -10,6 +10,11 @@ export interface IListFile {
   key: string;
 }
 
+export interface IFileUploadeProgress {
```

Comment:
is it possible to fix the spelling of the interface name?

Reply:
Good catch: fixed